.github/workflows/publish.yml

```diff
@@ -97,7 +100,10 @@ jobs:
         working-directory: arrow-libs
         run: |
           ./gradlew publishToSonatype
-          echo "$(cat gradle.properties | grep VERSION_NAME | cut -d'=' -f2) MingwX64 deployed!"
+          echo "$(cat gradle.properties | grep projects.version | cut -d'=' -f2) MingwX64 deployed!"
+
+      - name: Stop Gradle daemons
+        run: ./gradlew --stop
```

Comment:
This was to fix the cache issue? Is there any link you can reference? Curious about the technical details :p

Reply:
I think it was a workaround in the issue filed in the Cache repo, this one https://github.com/actions/cache/issues/454.

Not sure why, I guess the daemon blocks the access to those folders and then it fails, similar what you get in local when you run a lot of Gradle tasks over the same project.

Reply:
Thanks for the update @JavierSegoviaCordoba!